app/components/CardBet.tsx

```diff
@@ -1,39 +1,71 @@
 "use client";
 
 import { Card } from "@/app/components/ui";
-import { FixedProductMarketMaker } from "@/queries/omen";
+import { UserPosition } from "@/queries/conditional-tokens/types";
+import { FixedProductMarketMaker, getConditionMarket } from "@/queries/omen";
 import { remainingTime } from "@/utils/dates";
+import { useQuery } from "@tanstack/react-query";
 import { cx } from "class-variance-authority";
 import Link from "next/link";
 import { Button, Logo, Tag } from "swapr-ui";
 
-interface CardBetProps {
-  market: FixedProductMarketMaker;
+interface BetProps {
+  userPosition: UserPosition;
 }
 
-export const CardBet = ({ market }: CardBetProps) => {
+export const CardBet = ({ userPosition }: BetProps) => {
+  const conditionId = userPosition.position.conditionIdsStr;
+
+  const { data, isLoading, isFetching } = useQuery({
+    queryKey: ["getConditionMarket", conditionId],
+    queryFn: async () =>
+      getConditionMarket({
+        id: conditionId,
+      }),
+    enabled: !!conditionId,
+  });
+
+  if (isLoading || isFetching) return <LoadingCardBet />;
+
+  const market: FixedProductMarketMaker = data?.conditions[0]
+    ?.fixedProductMarketMakers[0] as FixedProductMarketMaker;
+
   const closingDate = new Date(+market.openingTimestamp * 1000);
+  const answer = market.question?.currentAnswer;
```

Comment:
This value need to be converted into an integer. Use `fromHex` from viem to do this.

```suggestion
  const answer = fromHex(
      market.question?.currentAnswer,
      "number"
    );
```

Answer will the winner index. Which we can compare to `outcomeIndex - 1`